backend/climateconnect_main/urls.py

```diff
@@ -96,8 +96,4 @@
     path('api/', include('ideas.urls')),
 ] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
-if settings.DEBUG == "True":
```

Comment:
https://github.com/climateconnect/climateconnect/blob/437a220568b3443067ddf4f979536246516cb76d/backend/climateconnect_main/settings.py#L32

`settings.DEBUG` is a bool (as it should be) so this condition has never been true. You should've used `if settings.DEBUG:` 

All the best :)